src/i18n/en.js

```diff
@@ -762,8 +762,10 @@ With Doppler you can generate Segments with common interests, create Forms to at
   },
   upgrade_suggestion_form: {
     breadcrumb: 'Plans',
+    breadcrumb_increase_plan: 'Increase Plan',
+    breadcrumb_maximum_plan: 'Plan Maximum',
     breadcrumb_url: `${urlBuyMonthly}`,
-    description: `Tell us about your needs and we'll create the perfect Plan for you.`,
+    description: `Our sales team will advice you about the most convinient Plan according to your needs.`,
```

Comment:
```suggestion
    description: `Our sales team will advice you about the most convenient Plan according to your needs.`,
```
https://dictionary.cambridge.org/dictionary/english/convenient